src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,106 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    public ArrayList<Task> list;
```

Comment:
Distinguish clearly between single-valued and multi-valued variables. If possible, you can name the list into other more meaningful and obvious names like tasks.